utils/cli/multisend.go

```diff
@@ -366,7 +367,7 @@ func NewQueryTotalGasCmd() *cobra.Command {
 			authquerier := authtypes.NewQueryClient(clientCtx)
 
 			account := args[0]
-			chainid := args[1]
+			// chainid := args[1]
```

Comment:
remove this comment

Reply:
done. see https://github.com/lavanet/lava/commit/5b44b0fc38bc2ab755612c85c83be4c287d25e9d